assets/scss/styles.scss

```diff
@@ -53,14 +79,14 @@ h1 {
 }
 
 /* Dashboard */
-#centres {
+.centre_data {
   padding: 0;
   h2 {
     padding-bottom: $padding+5;
     line-height: 24px;
```

Comment:
Tom and I agree that using fixed size is not accessible. This was based on Karwai's designs at the time. Will be revising.